x/bank/atlas.toml

```
@@ -0,0 +1,34 @@
+[module]
+description = "The bank module is responsible for handling multi-asset coin transfers between accounts and tracking special-case pseudo-transfers which must work differently with particular kinds of accounts."
+homepage = "https://github.com/cosmos/cosmos-sdk"
+keywords = [
+  "tokens",
+  "bank",
+  "transfer",
+  "asset",
+  "fungible",
+  "coins",
+]
+
+name = "x/bank"
+
+[bug_tracker]
+url = "https://github.com/cosmos/cosmos-sdk/issues"
+
+[[authors]]
+name = "alexanderbez"
+
+[[authors]]
+name = "fedekunze"
+
+[[authors]]
+name = "cwgoes"
+
+[[authors]]
+name = "alessio"
+
+[version]
+documentation = "https://raw.githubusercontent.com/cosmos/cosmos-sdk/v0.39.1/x/bank/atlas.md"
```

Shouldn't the documentation point to actual bank docs?
```suggestion
documentation = "https://raw.githubusercontent.com/cosmos/cosmos-sdk/v0.39.1/x/bank/spec/README.md"
```

---

No, it points to atlas-specific, and more importantly, the version-specific document.

Remember, the documentation that is displayed on Atlas:

- Must contain no relative links (which SPECs naturally have)
- Should be geared towards high-level gist of bootstrapping, CLI and REST usage and maybe a FAQ

---

Once I write the doc (above), I think it'll make more sense.